help_to_heat/frontdoor/epc_api.py

```diff
@@ -8,21 +8,28 @@
 
 
 class EPCApi:
+    def __init__(self):
+        self.base_url = settings.OPEN_EPC_API_BASE_URL
```

Comment:
Can we just use `settings.OPEN_EPC_API_BASE_URL` directly?

Reply:
sure